cukinia

```diff
@@ -196,6 +206,26 @@ _cukinia_http_request()
 	wget -q -O /dev/null "$url"
 }
 
+# _cukinia_mount: check for a specific mount point
+# arg1: the mount source
+# arg2: the mount destination
+# arg@: (optinal) a list of options to look for
+_cukinia_mount()
+{
+	local source="$1"; shift
+	local mount_point="$1"; shift
+	local re=""
```

Comment:
you don't need to set a variable to empty string in shell, it's its default value